README.md

```diff
@@ -1,10 +1,35 @@
-# EU eHealthNetwork Digitial COVID Certificate value sets
+# EU eHealthNetwork Digital COVID Certificate value-sets
 
-This repository contains the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema).
+Welcome to the EU-DCC value-sets repository. This repository contains a snapshot of the value-sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema). The most up-to-date version of the value-sets are distributed on the DCCG (DCC Gateway).
+
+All of the file found in this repository should be considered for reference usage only. The current official release is published on the EU DGC Gateway. As access to this gateway is only available to Member States please refer to your own Member State to for access to those files.
 
 Release: 2.11.0
 
-Note that these value sets do not form a core part of the [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema) but are referenced by it. These value sets can (and should) be regularly updated and distributed from the main Digital COVID Certificate Gateway (DCCG), under clear versioning and release management.
+## Overview
+
+Here is an overview of the files available in this repository. The column "managed on" describes the place where the value-set is managed. In most cases that is in this Github repository. Those value-sets will usually be very close to the versions provided on the EU DGC Gateway.
+
+**Reminder: the value-sets on the EU DGC Gateway are leading, the value-sets here should only be used for development purposes.**
```

Comment:
Apart from the RAT stuff, this repo is (effectively) leading through process. Maybe we can reflect that?

Also: "EU D*C*C Gatewau", right?

Reply:
It's called the EU DGC Gateway, https://github.com/eu-digital-green-certificates/dgc-gateway. I'm keeping it consistent with that, as when you google "EU DGC Gateway" you get the gateway and "EU DCC Gateway" gives the DCC docs.

Reply:
The 1st paragraph speaks about "EU DCC Gateway": let's change that for consistency. Agree with sticking to the leading sources.